docs/source/index.rst

```diff
@@ -10,14 +10,25 @@ the detailed API documentation and extensive examples. Check out :doc:`getting_s
 
```

Comment:
Maybe add a note saying that the documentation needs to be completed and invite anyone who wants to contribute?

```suggestion
..note: 
    This is a preliminary version of the DashAI documentation. 
    There are still several pages to be written. If you want to contribute to the documentation or the project, open a pr in the `project repository <https://github.com/DashAISoftware/DashAI>`_ and feel free to ask any questions you may have!
```